### ISSUE TITLE
Title label and VoiceOver

### ISSUE THREAD

**Issue by PatrickMounteney** [appcelerator-modules/titanium-web-dialog#4]
I have a blind person using my app who using iOS's VoiceOver to navigate around it. He tells me that when I use ti.webdialog the title label (showing the website address) covers the 'Done' button and 'Reload' control. Interestingly the latter can be selected with a deft tap on it, but the 'Done' button is tricker to select. I think I'm right that on other views where there is a title the label only extends to the bounds of the text it contains. On ti.webdialog (as viewed by a sited person) the label extends right across the screen. 
One other thing - I can't seem to get the 'title' property to work when doing webDialog.open - once opened the web dialog always shows the website address.

**Comment by hansemannn** [comment]
I'm afraid this is an Apple limitation. The API's being able to be exposed are limited and I cannot find any API's that relate to accessibility features there so far.

**Comment by PatrickMounteney** [comment]
> I'm afraid this is an Apple limitation. The API's being able to be exposed are limited and I cannot find any API's that relate to accessibility features there so far.

OK thanks Hans - it was worth a shot.
